bt-target/client/main.lua

```diff
@@ -86,7 +91,13 @@ function playerTargetEnable()
 
                                 success = true
 
-                                SendNUIMessage({response = "validTarget", data = Zones[_]["targetoptions"]["options"]})
+                                local options = Zones[_]["targetoptions"];
```

Comment:
Same down here, have you tested these? I'm not sure this will even run

Reply:
Why wouldn't it?

Reply:
I use semicolons all over my code because im used to them. It is probably not a standard but there is no reason why they would be invalid here.

Reply:
As mentioned by @dhawton it's best to just keep the syntax the same across the whole project